.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    root: true,
+    parser: '@typescript-eslint/parser',
+    plugins: [
+        '@typescript-eslint'
+    ],
+    extends: [
+        // 'eslint:recommended',
+        'airbnb-typescript/base',
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/recommended'
+    ],
+    parserOptions: {
+        project: './tsconfig.json'
+    },
+    rules: {
```

Comment:
@MartinKristof Are there some other rules you would recommend or some standard rules we use in LMC?

Reply:
airbnb-base

Reply:
added yet